app/views/admin/calendar/_calendar.slim

```diff
@@ -2,6 +2,6 @@ section.calendar
   h1.sr-only = t(:calendar)
   = render "admin/calendar/legend"
       
-  div#calendar.loan-calendar
+  div#calendar.loan-calendar data-loan-id="loan-#{loan.id}"
```

Comment:
Why do you need `loan-` as a prefix to the loan ID. Seems redundant? Can you remove, and then you wouldn't need `replace` in the JS?